release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2756,3 +2756,27 @@ link:https://access.redhat.com/solutions/6906441[{product-title} 4.9.28 containe
 ==== Updating
 
 To update an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[[ocp-4-9-29]]
+=== RHSA-2022:1363 - {product-title} 4.9.29 bug fix and security update
+
+Issued: 2022-04-20
+
+{product-title} release 4.9.29, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:1363[RHSA-2022:1363] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:1362[RHBA-2022:1362] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6934361[{product-title} 4.9.29 container image list]
+
+[id="ocp-4-9-29-bug-fixes"]
+==== Bug fixes
+* Previously, the Local Storage Operator (LSO) added an `OwnerReference` object to the persistent volumes (PV) it created, which sometimes caused an issue where a delete request for a PV could leave the PV in the `terminating` state while still attached to the pod. With this update, the LSO no longer creates an `OwnerReference` object and cluster administrators are now able to manually delete any unused PVs after a node is removed from the cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2070617[*BZ#2070617*])
```

Comment:
Previously, the Local Storage Operator (LSO) added an `OwnerReference` object to the persistent volumes (PV) it created, which sometimes caused an issue where a delete request for a PV could leave the PV in the `terminating` state while still attached to the pod. With this update, the LSO no longer creates an `OwnerReference` object, and cluster administrators can delete unused PVs after a node is removed from the cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2070617[*BZ#2070617*])